week-3/A-array-find/exercise.js

```diff
@@ -5,7 +5,9 @@
 */
 
 // -- Complete this function -->
-function findLongNameThatStartsWithA(names) {}
+function findLongNameThatStartsWithA(names) {
```

Comment:
This is written as a predicate function which just checks the required conditions for a single name. So `findLongNameThatStartsWithA` is no longer a good choice of name since it doesn't do any finding itself. Similarly the parameter called  `names` is actually only a single `name` so should be renamed :slightly_smiling_face: .

Reply:
sure I will commit the change